### PR TITLE
SEO-311 Fix pagination for Insights after Paginator class refactor

### PR DESCRIPTION
**Pull request by kamilkoterba**
https://wikia-inc.atlassian.net/browse/CE-3747

While code reviewing [Paginator changes](https://github.com/Wikia/app/pull/10507/files) found out that InsightsPaginator won't work.

Here's the fix.

@Wikia/spitfires @gabrys

### CODE REVIEW

**Comment by lukaszkonieczny** [comment]
LGTM

File: extensions/wikia/InsightsV2/helpers/InsightsPaginator.php [hunk 1]
```diff
@@ -12,13 +12,16 @@ class InsightsPaginator {
...
-	private $page = 0;
+	private $page = 1;
 	private $subpage;
 	private $params = [];
 
 	public function __construct( $subpage, $params ) {
 		$this->subpage = $subpage;
-		$this->params = $params;
+		if ( isset( $params['page'] ) ) {
+			$this->page = $params['page'];
```
Comment (gabrys) [inline extensions/wikia/InsightsV2/helpers/InsightsPaginator.php:22]: You could `unset( $params['page'] )` here and remove `filterParams` method.

**Comment by gabrys** [comment]
Thanks @kamilkoterba @lukaszkonieczny

**Comment by gabrys** [comment]
LGTM

**Pull request merged**